projects/commudle-admin/src/app/components/search-bar/search-bar.component.html

```diff
@@ -0,0 +1,31 @@
+<div (clickOutside)="toggleSearchSuffix(false, $event)" appClickOutside class="home-navbar-search">
+  <div class="d-flex search-container">
+    <nb-form-field (click)="toggleSearchSuffix(true)" class="home-navbar-search-form">
+      <nb-icon icon="search" nbPrefix></nb-icon>
+      <input (ngModelChange)="searchQueryChanged.next($event)" [ngModel]="searchQuery" nbInput
+             placeholder="Search events, labs, communities, people" type="text">
+    </nb-form-field>
+    <nb-select (selectedChange)="getSelectedFilters($event)" class="search-filter" multiple nbSuffix
+               placeholder="Filters">
+      <nb-option *ngFor="let filter of filterBy" [value]="filter">{{filter | titlecase}}</nb-option>
+    </nb-select>
+  </div>
+
+  <div class="home-navbar-search-suffix">
```

Comment:
we can just shorten the class to suffix or search suffix

Reply:
Done!